devops/pipeline/stages/source_build.yaml

```diff
@@ -56,10 +56,10 @@ stages:
       displayName: Generate build
 
     - script: |
-        docker exec $(CONTAINER_ID) bash -c "cmake --build /AzOsConfig/build && cmake --build /AzOsConfig/build --target dist && mkdir /staging && cp -r /AzOsConfig/build/dist/* /staging"
+        docker exec $(CONTAINER_ID) bash -c "cd /AzOsConfig/build && cmake --build . && cmake --build . --target dist && mkdir staging && cp -r /AzOsConfig/build/dist/* staging"
```

Comment:
Is it possible to remove all instances of `AzOsConfig`? Will this break our pipelines to change it to `azure-osconfig`?

Reply:
i rather do that separately. Too many changes for just that reference change